railties/lib/rails/application/configuration.rb

```diff
@@ -409,6 +422,16 @@ def annotations
         Rails::SourceAnnotationExtractor::Annotation
       end
 
+      # Configures the Content-Security-Policy:
+      #
+      #   config.content_security_policy do |policy|
+      #     policy.default_src :self, :https
+      #     policy.font_src    :self, :https, :data
+      #     policy.img_src     :self, :https, :data
+      #
+      #     # Specify URI for violation reports
+      #     policy.report_uri "/csp-violation-report-endpoint"
+      #   end
```

Comment:
I feel like https://api.rubyonrails.org/v7.0.2/classes/ActionDispatch/ContentSecurityPolicy.html should do the heavy lifting for explaining this API, and this method should link to that doc.  Thoughts?

Reply:
Yes, good point. I'm also documenting the other one.

Reply:
I'm linking to ActionDispatch::ContentSecurityPolicy now.